libvmi/driver/xen/xen_events.c

```diff
@@ -668,6 +669,9 @@ void xen_events_destroy_new(vmi_instance_t vmi)
     rc = xen->libxcw.xc_monitor_write_ctrlreg(xch, dom, VM_EVENT_X86_CR4, false, false, false);
     rc = xen->libxcw.xc_monitor_write_ctrlreg(xch, dom, VM_EVENT_X86_XCR0, false, false, false);
     rc = xen->libxcw.xc_monitor_software_breakpoint(xch, dom, false);
+    rc2 = xen_set_guest_requested_event(vmi, 0);
```

Comment:
Not really, it's safe to issue these anytime, even if they were not enabled previously.